gordon/metrics/log.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import time
+
+import zope.interface
+
+from gordon import interfaces
+
+# TODO: update plugin loader to load metrics logger as default
+
+
+@zope.interface.implementer(interfaces.ITimer)
+class LogTimer:
+    """Timer that outputs metrics to the logger.
+
+    Args:
+        metric (dict): Metric to output.
+        logger (.MetricLogger): Object used for outputting metrics.
+        time_unit (number): (optional) Multiplier used to scale time.time()
```

Comment:
I'd like some clarity here too (i.e. "`1` is seconds, `1000` for milliseconds, etc), and maybe mention that it defaults to `1`

Reply:
Sorry to harp on this again: `time_unit` to me translates to `ns`, `ms`, `s`, etc. Which I feel might be more common in general. Would there be any issue to switching to that?

Reply:
@econchick I'm fine with adding a bit more logic to switch between ns, ms, and s. We should apply the same approach to the ffwd module though for consistency, but maybe in the next PR to not block #22?